pkg/corerp/renderers/container/render.go

```diff
@@ -109,6 +109,20 @@ func (r Renderer) GetDependencyIDs(ctx context.Context, dm v1.DataModelInterface
 		}
 	}
 
+	for _, envvars := range properties.Container.Env {
+		if envvars.ValueFrom != nil && envvars.ValueFrom.SecretRef != nil {
+			if strings.HasPrefix(envvars.ValueFrom.SecretRef.Source, "/") {
+				resourceID, err := resources.ParseResource(envvars.ValueFrom.SecretRef.Source)
+				if err != nil {
+					return nil, nil, v1.NewClientErrInvalidRequest(fmt.Sprintf("invalid source: %s. Must be either a kubernetes secret name or a valid resourceID", envvars.ValueFrom.SecretRef.Source))
+				}
+				if resources_radius.IsRadiusResource(resourceID) {
```

Comment:
If it is not a Radius resource, what do we do? Do we just ignore it?

Reply:
+1 shouldn't it error out if it happens?